<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Demonstration</class>
 <widget class="QWidget" name="Demonstration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="BigButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>450</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LittleButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>480</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LiftButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>510</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>向左</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RightButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>540</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>向右</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UpButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>570</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>向上</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DownButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>600</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>向下</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ResetButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>210</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>还原</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>240</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>280</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Return">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>300</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>360</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存至文件</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ShowPeople">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>420</y>
     <width>88</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>显示人</string>
   </property>
  </widget>
  <widget class="QLabel" name="Day">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>第1天</string>
   </property>
  </widget>
  <widget class="QLabel" name="Day_Time">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>0时0分</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Start">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>150</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pause">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>180</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SpeedUp">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>90</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
  </widget>
  <widget class="QSlider" name="Speed">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>90</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>900</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QLabel" name="one">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>健康：</string>
    </property>
   </widget>
   <widget class="QLabel" name="two">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>感染者：</string>
    </property>
   </widget>
   <widget class="QLabel" name="three">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>重症患者：</string>
    </property>
   </widget>
   <widget class="QLabel" name="five">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>已接种疫苗：</string>
    </property>
   </widget>
   <widget class="QLabel" name="four">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>死亡：</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
